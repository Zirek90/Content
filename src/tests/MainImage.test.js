import React from "react";
import renderer from "react-test-renderer";
import MainImageComponent from "../components/MainImage";

describe("MainImageComponent", () => {
  it("renders correctly", () => {
    const props = {
      leadImage: {
        url:
          "/859f2008-a40a-4b92-afd0-24bb44d10124/dxresources/0874/0874022e-89fe-448e-af2f-d7a88b7baa04.jpg",
      },
      leadImageCaption: {
        value: "Listen your voice",
      },
    };
    const url = "https://content-eu-4.content-cms.com";
    const tree = renderer
      .create(<MainImageComponent mainImage={props} url={url} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
