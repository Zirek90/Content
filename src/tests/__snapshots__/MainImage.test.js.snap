// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainImageComponent renders correctly 1`] = `
<figure
  className="sc-dlnjPT iicaHi"
>
  <img
    alt="Listen your voice"
    className="sc-iCoHVE dAOolK"
    src="https://content-eu-4.content-cms.com/859f2008-a40a-4b92-afd0-24bb44d10124/dxresources/0874/0874022e-89fe-448e-af2f-d7a88b7baa04.jpg"
  />
  <figcaption>
    Listen your voice
  </figcaption>
</figure>
`;
